File: source/zombie/audio/DummySoundBank.java
// Decompiled on Sat Jan 17 08:24:00 EST 2026 with Zomboid Decompiler v0.2.3 using Vineflower.
package zombie.audio;

import fmod.fmod.FMODFootstep;
import fmod.fmod.FMODVoice;

public class DummySoundBank extends BaseSoundBank {
    @Override
    public void addVoice(String alias, String sound, float priority) {
    }

    @Override
    public void addFootstep(String alias, String grass, String wood, String concrete, String upstairs) {
    }

    @Override
    public FMODVoice getVoice(String alias) {
        return null;
    }

    @Override
    public FMODFootstep getFootstep(String alias) {
        return null;
    }
}

File: converted2/src/zombie/audio/DummySoundBank.cpp
#include "zombie/audio/DummySoundBank.h"

namespace zombie {
namespace audio {

void DummySoundBank::addVoice(const std::string &alias,
                              const std::string &sound, float priority) {
  // TODO: Implement addVoice
}

void DummySoundBank::addFootstep(const std::string &alias,
                                 const std::string &grass,
                                 const std::string &wood,
                                 const std::string &concrete,
                                 const std::string &upstairs) {
  // TODO: Implement addFootstep
}

FMODVoice DummySoundBank::getVoice(const std::string &alias) {
  // TODO: Implement getVoice
  return nullptr;
}

FMODFootstep DummySoundBank::getFootstep(const std::string &alias) {
  // TODO: Implement getFootstep
  return nullptr;
}

} // namespace audio
} // namespace zombie
